exercises/say/canonical-data.json

```diff
@@ -61,6 +61,12 @@
       "expected": "one thousand two hundred thirty-four"
     },
     {
+      "description": "eight hundred and ten thousand",
+      "property": "say",
+      "input": 810000,
+      "expected": "eight hundred and ten thousand"
```

Comment:
The **and** needs to be removed as this is only part of the extension. The base exercise ignores the and part.